openverse_catalog/dags/providers/provider_api_scripts/inaturalist.py

```diff
@@ -144,6 +143,73 @@ def compare_update_dates(
         raise AirflowSkipException("Nothing new to ingest")
 
     @staticmethod
+    def load_catalog_of_life_names():
+        COL_URL = "https://api.checklistbank.org/dataset/9840/export.zip?format=ColDP"
+        # Tried importing common.loader.paths.STAGING_DIRECTORY but it didn't work in
+        # local environment, TO DO: find a better place for this process.
+        DATA_DIR = Path(__file__).parents[4]
```

Comment:
Ah, we no longer have `STAGING_DIRECTORY` there! Best place for it might be [`OUTPUT_DIR`](https://github.com/WordPress/openverse-catalog/blob/b4ef93ce5f825e4ae0e3ba3814340ec42ade7fa2/docker/airflow/Dockerfile#L23)? Although would we want to make sure this gets cleaned up, or have it stick around for the lifetime of the container (e.g. however long it is until the next deployment)? I see you have a note about getting this into S3, would that be more resilient?

Reply:
Yeah, I'm not sure. I did a project once that would download tables via an API (kind of like the Catalog of Life data), and store them on S3 under a dated prefix, just because I was a hard core data hoarder, and then Redshift would pretty much only have the current day's data except for very special cases where if we wanted to know when something changed it wasn't impossible to go back through the archive on S3. But I think that's might overkill for here. (?) Maybe it would make sense to have something like a `last_inaturalist_data_load/` prefix in the bucket where the tsv files normally go, where the raw source files could go? And yeah, changing this to `OUTPUT_DIR` sounds great.

Reply:
I could see that working! Especially with the cadence this DAG will run at, putting this where we'd store the other data (either TSV or otherwise) sounds like a good call 🙂